recipes/openms-meta/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+qt:
```

Comment:
not sure if this has an effect. We are using qt-main now in the package.

Reply:
according to the log it uses 5.15 as on conda-forge

Reply:
yes because qt-main is not pinned by bioconda.

Reply:
ok should I remove it this customization?

Reply:
either this or you change to qt-main for the future (in case pinnings get messed up again)

Reply:
but maybe better to remove in case another pinned pacakge like boost suddenly becomes incompatible with that version. I dont know. Bioconda people should not.